doc/api/dns.md

```diff
@@ -173,15 +173,13 @@ Valid values for `rrtype` are:
  * `'SOA'` - start of authority record
  * `'NAPTR'` - name authority pointer record
 
-The `callback` function has arguments `(err, addresses)`. When successful,
-`addresses` will be an array, except when resolving an SOA record which returns
-an object structured in the same manner as one returned by the
-[`dns.resolveSoa()`][] method. The type of each item in `addresses` is
-determined by the record type, and described in the documentation for the
-corresponding lookup methods.
+The `callback` function has arguments `(err, addresses)` and will receive an
```

Comment:
Nit: I would retain the ". When successful, `addreses` will be" wording. The revised text says the callback "will receive an array" but it's not immediately clear that the array is `addresses`. That was clear in the previous wording.